00-sql-introduction.md

```diff
@@ -78,7 +85,15 @@ In particular, we want to use a tool where it's easy to repeat our analysis
 in case our data changes.  We also want to do all this searching without 
 actually modifying our source data.  
 
-Putting our data into a database and using SQL will help us achieve these goals.  
+Putting our data into a relational database and using SQL will help us achieve these goals.  
+
+> ### Definition: _Relational database_
+>
+> A relational database is a digital database where all data is stored in relations (or tables)
+> containing rows and columns. All tables contain records and all records are identified by a
+> field contiaing a unique value. Every table shares at least one field with another table in
+> one-to-one, one-to-many or many-to-many relationships. This allows the user to access the data
+> in many ways 
```

Comment:
I'm wondering whether there's a way to structure this definition in a way that doesn't give the learner as many pieces of jargon (rows, columns, relations, records, value) in one paragraph. Do you have a concept map we could use to guide us as we structure this?

Should we reuse the word "database" in "digital database" or could we use a phrasing like "data storage application"?

You should probably add a period at the end of the last sentence.